src/main/java/org/jenkinsci/plugin/Diskcheck.java

```diff
@@ -10,195 +11,180 @@
 import hudson.tasks.BuildWrapper;
 import hudson.node_monitors.*;
 import hudson.model.Node;
-import hudson.model.Result;
-import hudson.tasks.BatchFile;
-import hudson.tasks.CommandInterpreter;
-import hudson.tasks.Shell;
 
 import org.kohsuke.stapler.DataBoundConstructor;
 
-import java.io.File;
 import java.io.IOException;
 import java.io.PrintStream;
-
-
+import java.net.UnknownHostException;
+import java.util.List;
 
 /**
  * Class to allow any build step to be performed before the SCM checkout occurs.
- * 
+ *
  * @author Manoj Thakkar
+ * @author Frano Stanisic (8th Decemeber 2015, updated deletion to use Jenkins API/only delete files up to a threshhold)
```

Comment:
In general, I'd advise to _not_ add those tags. IMO they're not useful since they become out-of-date very quickly. And SCM is a better tool to know what changed in a source file.